selftests/inspekt-style.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh -e
-inspekt style . --exclude=.git --disable E501,E402,E722
+inspekt style . --fix --exclude=.git --disable E501,E402,E722
```

Comment:
IIUC, this is a fix in place that is inside a selftest/ directory. So I'm running a test and suddenly my code is changed. Not sure about this here.

What about adding a utility to fix this on scripts/?